src/assets/scss/_array-field.scss

```diff
@@ -0,0 +1,135 @@
+.squished-array-buttons {
+  .array-input-field-container {
+    .add-button {
+      padding-right: 8px;
+    }
+    .input-items-new {
+      .item {
+        .action-buttons {
+          margin-right: -1px;
+        }
+      }
+    }
+  }
+}
+
+// New Styles
+.entity-value,
+.select-all-input,
+.multi-form-input {
+  .add-button {
+    padding-right: 6px;
+  }
+  .array-input-field-container {
+    ul {
+      padding-left: 0;
+    }
+    .add-element-container {
+      display: flex;
+      flex-direction: row;
+      gap: 5px;
+      [class*='input-switch-container'] {
+        width: 100%;
+      }
+      .chaise-btn {
+        height: 30px;
+      }
+      &.add-margin-top {
+        margin-top: 10px;
+      }
+    }
+    .input-items-new {
+      width: 100%;
+      margin-bottom: 0px;
+      display: flex;
+      flex-direction: column;
+      gap: 3px;
+
+      & *:focus {
+        outline: none;
+      }
+      li {
+        list-style: none;
+      }
+      .item {
+        border-radius: 3px;
+        min-height: 30px;
+        height: auto;
+        width: 100%;
+        padding-top: 1px;
+        padding-bottom: 1px;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        gap: 5px;
+        &:hover {
+          box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+        }
+        .move-icon {
+          display: flex;
+          padding-left: 3px;
+          & * {
+            position: relative;
+            top: 50%;
+            transform: translateY(-7px);
+          }
+        }
+        .input-box,
+        .input-switch {
+          border: 2px solid transparent;
```

Comment:
What's the point of this border and box-shadow? It's adding unnecessary padding and when the input is displayed you can see it:

![Screenshot 2024-04-22 at 5 41 56 PM](https://github.com/informatics-isi-edu/chaise/assets/2129750/0cb45bc0-c64f-4c27-ae88-34ad4e5e7cf0)

This is not related to this rule but since I added the screenshot to this comment, I'm mentioning it here: Please remove the `height: 100%;` rule on `_recordedit.scss` line 128. With this rule the top border of each cell is not showing up properly when you activate the "select all/some" feature (see how the second form's height is smaller than it should be).

Reply:
I removed those rules since they are no longer required. They were a part of a previous design.